drivers/serial/uart_nrfx_uart.c

```diff
@@ -932,18 +930,17 @@ static int uart_nrfx_init(struct device *dev)
 
 #if defined(CONFIG_UART_0_ASYNC) || defined(CONFIG_UART_0_INTERRUPT_DRIVEN)
 
-	IRQ_CONNECT(DT_NORDIC_NRF_UART_UART_0_IRQ_0,
-		    DT_NORDIC_NRF_UART_UART_0_IRQ_0_PRIORITY,
+	IRQ_CONNECT(UART_0_IRQ,
+		    DT_IRQ(UART_0, priority),
```

Comment:
Mixing direct calls and macros (such as UART_0_IRQ) in small portion of code feels confusing.
Might be better to avoid that

Reply:
> Mixing direct calls and macros (such as UART_0_IRQ) in small portion of code feels confusing.

Sure. I was avoiding defining a macro and only using it once, but I agree with you.